package.json

```diff
@@ -19,22 +19,28 @@
     "lib"
   ],
   "main": "index.js",
+  "type": "module",
   "engines": {
-    "node": ">=8.6"
+    "node": ">=12"
```

Comment:
The version 12 of NodeJS is the first to support ESM without experimental flags. 
The version 10 can still be supported (see work with bundler below). 
Lower versions can also be supported but they are not supported anymore (see [schedule](https://nodejs.org/en/about/releases/)).

I don't know if I should fill that field with the lowest supported version (8?), the lowest supported version without bundling but allowing experimental flag (10?) or the lowest supported version without experimental flag (12?).